runbotics/runbotics-scheduler/src/utils/mergeArrays.ts

```diff
@@ -0,0 +1,6 @@
+export const mergeArrays = (valuesToMerge: unknown[]) => {
+    const uniqueValues = new Set(valuesToMerge);
+    const nowDuplicatesArray = Array.from(uniqueValues);
+
+    return nowDuplicatesArray;
+};
```

Comment:
tbh I would expect it to accept 2 arrays as parameters and merge them together. What you do here is to delete duplications